platform-modules/batch-models/src/main/scala/org/ekstep/analytics/job/report/StateAdminGeoReportJob.scala

```diff
@@ -81,7 +81,7 @@ object StateAdminGeoReportJob extends optional.Application with IJob with StateA
     val window = Window.partitionBy("slug").orderBy(asc("districtName"))
     val blockDataWithSlug = blockData.
       select("*")
-      .groupBy(col("slug"),col("District name").as("districtName")).
+      .groupBy(col("slug"),col("District Name").as("districtName")).
```

Comment:
This change is less useful - maybe just 100% truly compliant to PRD, but also notice that all other columns have lower case letter (example, Block id). I don't recommend these changes now. Please restrict to adding district names in the validated-user section alone.

Reply:
reverted to minimal changes